src/main/java/com/twilio/oai/TwilioTerraformGenerator.java

```diff
@@ -307,7 +338,7 @@ private TerraformSchema buildTerraformSchema(final CodegenParameter codegenParam
 
     private TerraformSchema buildTerraformSchema(final CodegenProperty codegenProperty,
                                                  final TerraformSchemaOptions schemaOptions) {
-        return buildTerraformSchema(codegenProperty.dataType, schemaOptions, codegenProperty.items);
+        return buildTerraformSchema(codegenProperty.datatypeWithEnum, schemaOptions, codegenProperty.items);
```

Comment:
What's the difference between `dataType` and `dataTypeWithEnum`? and why isn't this a problem with the parameters?

Reply:
I tried using `baseType`, and found that it did not work with lists. A list of strings, for example, would have a `baseType` of `string`

Reply:
Weird. I thought lists will have a `baseType` of `array`